SpecTools/Classes/Action/Action+UIGestureRecognizer.swift

```diff
@@ -98,14 +98,13 @@ public typealias TargetActionInfo = [(target: AnyObject, action: Selector)]
         }
 
         /// Executes all targets on a specific gesture recognizer
-        @discardableResult public func execute() -> Action {
+        @discardableResult public func execute(_ recognizer: UIGestureRecognizer) -> Action {
```

Comment:
```suggestion
        @discardableResult public func execute(_ recognizer: UIGestureRecognizer? = nil) -> Action {
```

This won't break the interface which is public. If we force it to include the recogniser I will have to bump up the minor version. Otherwise nice add, happy to merge with this change.